### PR TITLE
report(redesign): pwa icons

### PR DESCRIPTION
**Pull request by Hoten**
Replaces the icons in the PWA subsections with the assets from https://github.com/GoogleChrome/lighthouse/pull/8222#issuecomment-483768116

ref #8185

### CODE REVIEW

File: lighthouse-core/report/html/report-styles.css [hunk 1]
```diff
@@ -136,13 +138,13 @@
...
+  --pwa-fast-reliable-gray-url: url('data:image/svg+xml;utf8,<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="M0 0h24v24H0z"/><circle fill-opacity=".15" fill="%23000" fill-rule="nonzero" cx="12" cy="12" r="10"/><path d="M12 6l5 2.222v3.334c0 3.083-2.133 5.966-5 6.666-2.867-.7-5-3.583-5-6.666V8.222L12 6zm-.446 10.222s.871-1.533 2.622-4.595l.031-.067c.04-.08.01-.227-.209-.227h-1.555l.444-3.11h-.444c-1.147 2.017-2.01 3.528-2.582 4.542-.01.017.053-.098-.032.053-.084.151-.088.293.17.293h1.555l-.445 3.111h.445z" fill="%23FFF" fill-rule="nonzero"/></g></svg>');
+  --pwa-installable-gray-url: url('data:image/svg+xml;utf8,<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="M0 0h24v24H0z"/><circle fill-opacity=".15" fill="%23000" fill-rule="nonzero" cx="12" cy="12" r="10"/><path d="M12 6c-3.312 0-6 2.688-6 6s2.688 6 6 6 6-2.688 6-6-2.688-6-6-6zm3 6.6h-2.4V15h-1.2v-2.4H9v-1.2h2.4V9h1.2v2.4H15v1.2z" fill="%23FFF" fill-rule="nonzero"/></g></svg>');
```
Comment (patrickhulce) [inline lighthouse-core/report/html/report-styles.css]: these are really hard to tell what's changed :) 

were these just taken directly from design?

Comment (Hoten) [reply lighthouse-core/report/html/report-styles.css]: from https://github.com/GoogleChrome/lighthouse/pull/8222#issuecomment-483768116 + optimized w/ https://jakearchibald.github.io/svgomg/

I plan on adding a step in our report generation that translates .svg files -> data urls in the report.

Comment (patrickhulce) [reply lighthouse-core/report/html/report-styles.css]: ah bingo!

File: lighthouse-core/report/html/report-styles.css [hunk 2]
```diff
@@ -136,13 +138,13 @@
...
+  --pwa-installable-color-url: url('data:image/svg+xml;utf8,<svg width="20" height="20" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="M-2-2h24v24H-2z"/><g fill-rule="nonzero"><circle fill="%230CCE6B" cx="10" cy="10" r="10"/><path d="M10 4c-3.312 0-6 2.688-6 6s2.688 6 6 6 6-2.688 6-6-2.688-6-6-6zm3 6.6h-2.4V13H9.4v-2.4H7V9.4h2.4V7h1.2v2.4H13v1.2z" fill="%23FFF"/></g></g></svg>');
+  --pwa-optimized-color-url: url('data:image/svg+xml;utf8,<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><rect fill="%230CCE6B" x="2" y="2" width="20" height="20" rx="10"/><path fill="%23FFF" d="M12 15.074l3.605 2.176-.957-4.1 3.185-2.76-4.194-.356L12 6.167l-1.64 3.867-4.193.356 3.185 2.76-.957 4.1z"/><path d="M0 0h24v24H0z"/></g></svg>');
```
Comment (brendankenny) [inline lighthouse-core/report/html/report-styles.css]: it seems like most of this is fill changes and stuff we don't need (`fill-rule` -- no need -- and relative positioning changes -- can just do with the sizing on `::before` -- and excess precision), maybe just keep the fill changes?

Comment (Hoten) [reply lighthouse-core/report/html/report-styles.css]: done

**Comment by Hoten** [comment]
image

**Pull request merged**